db/sql/revert/buildings/functions/building_outlines.sql

```diff
@@ -1,23 +1,334 @@
--- Revert nz-buildings:buildings/functions/building_outlines from pg
+-- Revert nz-buildings:buildings/functions/building_outlines to v1.0.4
 
 BEGIN;
 
-DROP FUNCTION buildings.building_outlines_insert(integer, integer, integer, integer, integer, integer, integer, geometry);
+--------------------------------------------
```

Comment:
I guess this should be `DROP FUNCTION`?

Reply:
No, this code sets it back to `v1.4.0` of the function, not back to nothing.

Reply:
I see what you meant this morning. The `sqitch rework` will copy the deploy file of the older version to revert :grimacing: